src/pages/[platform]/deploy-and-host/index.mdx

```diff
@@ -2,7 +2,7 @@ import { getChildPageNodes } from '@/utils/getChildPageNodes';
 import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
 
 export const meta = {
-  title: 'Deploy & host',
+  title: 'Deployment',
```

Comment:
Context as to why removing the `Host`?

Reply:
It was a decision after my discussion with @swaminator. The Host indicates more Web and only deployment can contain everything as a meaning.